tests/test_poster.py

```diff
@@ -7,7 +7,11 @@ class TestPoster(object):
     @classmethod
     def setup_class(cls):
         cls.port = free_port()
-        start_server(cls.port, locations={"data": "testData"})
+        start_server(cls.port, locations={"data": "testData"}, tables={"data": "testTables"})
+        cls.poster = Poster(url="http://localhost:{port}".format(port=cls.port))
```

Comment:
@Vatavuk in python3.6 we can just do this `f"http://localhost:{cls.port}"`

Reply:
@vladarefiev 👍